homeassistant/components/zwave/__init__.py

```diff
@@ -295,7 +295,7 @@ def value_added(node, value):
 
     def node_added(node):
         """Called when a node is added on the network."""
-        entity = ZWaveNodeEntity(node)
+        entity = ZWaveNodeEntity(node, NETWORK)
```

Comment:
We should not pass the network around. Preferably we migrate the `NETWORK` global to `hass.data[ZWAVE_NETWORK]`